### PR TITLE
added network layer to login

### PR DESCRIPTION
**Pull request by Pelozo**
**Sumary**: Added network layer to login

**Preview**: 
Grabación de pantalla 2021-09-20 a la(s) 12 11 41

**Trello card:** [[U2] Backend: Login](https://trello.com/c/0ZJO4HK

### CODE REVIEW

**Review by jcruz-wolox — APPROVED** [review]
Good job!

File: WoloxAndroidBootstrap/app/src/main/java/ar/com/wolox/android/example/ui/login/LoginFragment.kt [hunk 1]
```diff
@@ -52,6 +59,34 @@ class LoginFragment private constructor() : WolmoFragment<FragmentLoginBinding,
...
+        binding.loadingBar.visibility = View.GONE
+    }
+
+    override fun disableLoginButton() {
+        binding.loginButton.isEnabled = false
+    }
+
+    override fun enableLoginButton() {
+        binding.loginButton.isEnabled = true
+    }
```
Comment (ffasitella) [inline WoloxAndroidBootstrap/app/src/main/java/ar/com/wolox/android/example/ui/login/LoginFragment.kt]: You can put this in a single method like `toggleLoginButton(enabled: Boolean)`

File: WoloxAndroidBootstrap/app/src/main/java/ar/com/wolox/android/example/ui/login/LoginPresenter.kt [hunk 2]
```diff
@@ -1,32 +1,63 @@
...
+                dismissKeyboard()
+                showLoading()
+                disableLoginButton()
+            }
+            networkRequest(authRepository.getLogin(LoginRequest(user, password))) {
+                onResponseSuccessful {
+                    userSession.username = user
+                    view?.apply {
+                        dismissLoading()
+                        enableLoginButton()
```
Comment (ffasitella) [inline WoloxAndroidBootstrap/app/src/main/java/ar/com/wolox/android/example/ui/login/LoginPresenter.kt]: Every time you are calling `view?.dismissLoading()` and `view?.enableLoginButton()`, so you can remove this here and add it at the end of `networkRequest()`

**Review by ffasitella — CHANGES_REQUESTED** [review]
Just two little style comments!

**Review by ffasitella — APPROVED** [review]
Amazing 🙌🏽

**Pull request merged**